pkg/webhooks/regularuser/common/regularuser.go

```diff
@@ -263,28 +287,46 @@ func (s *RegularuserWebhook) authorized(request admissionctl.Request) admissionc
 
 // isMustGatherAuthorized check if request is authorized for MustGather CR
 func isMustGatherAuthorized(request admissionctl.Request) bool {
-	return slices.Contains(request.UserInfo.Groups, ceeGroup) &&
-		request.Kind.Kind == mustGatherKind &&
-		request.Kind.Group == mustGatherGroup
+	return slices.Contains(request.UserInfo.Groups, ceeGroup)
 }
 
 // isCustomDomainAuthorized check if request is authorized for CustomDomain CR
 func isCustomDomainAuthorized(request admissionctl.Request) bool {
-	return (slices.Contains(request.UserInfo.Groups, "cluster-admins") ||
-		slices.Contains(request.UserInfo.Groups, "dedicated-admins")) &&
-		request.Kind.Kind == customDomainKind &&
-		request.Kind.Group == customDomainGroup
+	return slices.Contains(request.UserInfo.Groups, "cluster-admins") ||
+		slices.Contains(request.UserInfo.Groups, "dedicated-admins")
 }
 
 // isNetNamespaceAuthorized check if request is authorized for NetNamespace CR
 func isNetNamespaceAuthorized(s *RegularuserWebhook, request admissionctl.Request) bool {
 	return (slices.Contains(request.UserInfo.Groups, "cluster-admins") ||
 		slices.Contains(request.UserInfo.Groups, "dedicated-admins")) &&
-		request.Kind.Kind == netNamespaceKind &&
-		request.Kind.Group == netNamespaceGroup &&
 		isNetNamespaceValid(s, request)
 }
 
+// isClusterVersionAuthorized only allows specific K8s serviceaccounts to modify ClusterVersion resources
+func isClusterVersionAuthorized(request admissionctl.Request) bool {
+	log.Info("analyzing clusterversion", "user", request.UserInfo.Username, "group", request.UserInfo.Groups)
```

Comment:
```suggestion
```
Is this debugging output? I'd suggest we remove this to remain consistent with the other `is*Authorized()` functions

Reply:
Ah, I thought it was useful to keep, but happy to remove it for consistency

Reply:
We could always do the inverse as well, and add more logging :slightly_smiling_face:

Reply:
I just took it out, not too attached to it